tests/testsuite/manifest.go

```diff
@@ -75,7 +75,7 @@ func ReadManifestYamlFile(pathToManifest string) []unstructured.Unstructured {
 	var objects []unstructured.Unstructured
 	stream, err := os.Open(pathToManifest)
 	util.PanicOnError(err)
-
+	defer stream.Close()
```

Comment:
We need to use `CloseIOAndCheckErr`

Reply:
You are correct,  I just do not think we need to be that strict in all scenarios.
I am not expecting anyone to go over the logs if tests pass. The only scenarios where this is a must, is in production code which may cause fd flood.

Considering we have a very large number of these already in the codebase, IMO we should accept such fixes and schedule to "fix" them as a follow up.

Reply:
Unfortunately, it was either this or changing the config of nogo. This is what has come to my mind at first.